src/_includes/side_nav.html

```diff
@@ -9,7 +9,7 @@
           <li class="related__section-link">
```

Comment:
I think with the two simplifications mentioned above (one in-line conditional per line; moving the first link out of the outer conditional), this abstraction might be less necessary